node/connection_base.js

```diff
@@ -266,6 +267,12 @@ TChannelConnectionBase.prototype.handleCallRequest = function handleCallRequest(
     }
 };
 
+TChannelConnectionBase.prototype.handlePingResponse =
```

Comment:
also this method should be on TChannelConnection, not this base class... handler entry points like this are to be out of base class